src/assets/HTMLAsset.js

```diff
@@ -73,14 +72,14 @@ const OPTIONS = {
 };
 
 class HTMLAsset extends Asset {
-  constructor(name, options) {
-    super(name, options);
+  constructor(name, pkg, options) {
+    super(name, pkg, options);
```

Comment:
We don't pass `pkg` to the `constructor` anymore for perf reasons : 

```js
constructor(name, options) {
  super(name, options)
```

Reply:
Updated.